school_library_decorator/rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
+end
```

Comment:
- [ ] In the class `Rental` the constructor of Rental should be modified so `Book` and `Person` are set in it to break the [many-to-many relationship](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship). 
    - The rental should be adding itself to the @book achieving the 'belongs-to' association
    `book.rentals << self`

    - The rental is adding itself to the @person the 'belongs-to' association
    `person.rentals << self`